sync-orders-to-bigQuery/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const { MoltinClient } = require('@moltin/request')
+const moment = require('moment')
+const datahelper = require('./bq-helper')
+const decryptHelper = require('./decrypt-helper')
+const mapping = require('./mapping')
+
+async function getUpdatedOrders(moltin, date, offset = 0, limit = 1) {
+  return new Promise(async (resolve, reject) => {
+    moltin.get(`orders?include=items&page[limit]=${limit}&filter=ge(updated_at,${date})&page[offset]=${offset}`)
+      .then(resolve)
+      .catch(reject)
+  })
+}
+
+module.exports.updateOrders = async (request, response) => {
+  try {
+    await datahelper.createBigQueryClient()
+    await decryptHelper.decryptVars()
+
+    const moltin = new MoltinClient({
+      client_id: process.env.MOLTIN_CLIENT_ID,
+      client_secret: process.env.MOLTIN_CLIENT_SECRET,
+    })
+
+    const date = moment().subtract(1, 'days').format('YYYY-MM-DD')
+
+    const limit = 100; let offset = 0; let resultSize = 0; let
+      result
+
+    do {
+      // Get Order Updates and delete older versions
+      const orders = await getUpdatedOrders(moltin, date, offset, limit)
```

Comment:
```suggestion
      const orders = await getUpdatedOrders(moltin, date, offset, limit)
      
```

Reply:
Again I can't see what this change does, maybe you could help me see :)